f/radio/fn_radioAddHandlers.sqf

```diff
@@ -0,0 +1,47 @@
+// F3 - Folk ARPS Long-Range Radio Module
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+
+/* ========================
+This function adds local event handlers to the player which grant radio channels when picking up a backpack or entering a vehicle. It also makes an initial check to see what they've got to begin with.
+It's activated by f\radio\f_radioChannels.sqf.
+=========================== */
+waitUntil {!(isNil "f_var_radioChannelsUnified")};
+
+_splitMode = f_var_radioSplitMode;
```

Comment:
I'm not sure if there's a guaranteed order of broadcast variables. So depending on how much faith we have in arma we could also wait for the `f_var_radioSplitMode` variable.

Reply:
also, `private`. and variable isn't even used?

Reply:
On mission start, the variables should be broadcast in order of instruction (at least when there's this much other stuff happening between instructions). On JIP, all public variables are sent before any scripts are run, so it's not a problem.

Reply:
_splitMode could be made private but I don't see any particular need to. It should always be the same in all contexts because it's a global setting.

f_var_radioChannelsUnified is not currently used (would have been used in an alternate version of the mode system) but I think it is potentially useful to expose for missionmakers.

Reply:
What I meant was that the variable `_splitMode` is never actually used anywhere in this file.

Reply:
I don't see why they wouldn't just use `f_var_radioSplitMode`